### ISSUE TITLE
CI fails with node 18.x

### ISSUE THREAD

**Issue by pierroberto** [facile-it/fortepiano#46]
```
##[debug]Evaluating condition for step: 'Run yarn test:coverage'
##[debug]Evaluating: success()
##[debug]Evaluating success:
##[debug]=> true
##[debug]Result: true
##[debug]Starting: Run yarn test:coverage
##[debug]Loading inputs
##[debug]Loading env
Run yarn test:coverage
##[debug]/usr/bin/bash -e /home/runner/work/_temp/4db9a911-40f7-49f[2](https://github.com/facile-it/fortepiano/runs/7425706316?check_suite_focus=true#step:5:2)-bed1-72189f65c7e[3](https://github.com/facile-it/fortepiano/runs/7425706316?check_suite_focus=true#step:5:3).sh
node:internal/errors:[4](https://github.com/facile-it/fortepiano/runs/7425706316?check_suite_focus=true#step:5:4)77
    ErrorCaptureStackTrace(err);
    ^
Error [ERR_LOADER_CHAIN_INCOMPLETE]: "file:///home/runner/work/fortepiano/fortepiano/.pnp.loader.mjs 'resolve'" did not call the next hook in its chain and did not explicitly signal a short circuit. If this is intentional, include `shortCircuit: true` in the hook's return.
    at new NodeError (node:internal/errors:387:[5](https://github.com/facile-it/fortepiano/runs/7425706316?check_suite_focus=true#step:5:5))
    at ESMLoader.resolve (node:internal/modules/esm/loader:8[6](https://github.com/facile-it/fortepiano/runs/7425706316?check_suite_focus=true#step:5:6)0:13)
    at async ESMLoader.getModuleJob (node:internal/modules/esm/loader:439:[7](https://github.com/facile-it/fortepiano/runs/7425706316?check_suite_focus=true#step:5:7))
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:541:24)
    at async loadESM (node:internal/process/esm_loader:[8](https://github.com/facile-it/fortepiano/runs/7425706316?check_suite_focus=true#step:5:8)3:5)
    at async handleMainPromise (node:internal/modules/run_main:65:[12](https://github.com/facile-it/fortepiano/runs/7425706316?check_suite_focus=true#step:5:13)) {
  code: 'ERR_LOADER_CHAIN_INCOMPLETE'
}
Node.js v[18](https://github.com/facile-it/fortepiano/runs/7425706316?check_suite_focus=true#step:5:19).6.0
Error: Process completed with exit code 1.
##[debug]Finishing: Run yarn test:coverage
```
See failed job [here](https://github.com/facile-it/fortepiano/runs/7425706914?check_suite_focus=true).

Here the [link](https://github.com/yarnpkg/berry/pull/4624) of the possible cause of the problem. This problem has been reported [here](https://issuemode.com/issues/yarnpkg/berry/106345684).

The problem looks related to the node version `18.x`. I tried to remove it from the list of `node-version` in `test.yaml`. Everything started to work again (see MR #45).

It's not clear if there is a different way to fix the problem. 

Any idea? @davidecaruso @xzhavilla

**Comment by xzhavilla** [comment]
Yarn related, imho let's just wait for an update.

By the way, I'm reconsidering the choice of Yarn due to incomplete support (e.g., Dependabot).

**Comment by pierroberto** [comment]
Ok, it looks that the problem has already been fixed but not released yet. It should work again soon🤞.

As for the package manager, I would prefer a better support over performances. I wouldn't mind switching to npm.

**Comment by pierroberto** [comment]
Fixed in #44
